### PR TITLE
[REVIEW][FEATRUE][GAWE-71] - Create service for complete profile

### PR DESCRIPTION
**Pull request by triandamai**
#### Description

> Complete profile used by step after otp confirmation.

#### Ticket Reference(s)

[GAWE-71](https://bluehabit.atlassian.net/browse/GAWE-71)

##### UI Changes

> Screenshot or video of UI-related changes

##### Coverage

> Test coverage screenshot.

N/A

#### Note(s)

> Mention other user or extra note unrelated to the MR here.

[GAWE-71]: https://bluehabit.atlassian.net/browse/GAWE-71?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Comment by triandamai** [comment]
@m-hari16 please Approve this PR after #8 and #9 merged, thanks

**Comment by m-hari16** [comment]
@triandamai  Looks good

File: src/main/java/com/bluehabit/eureka/services/SignUpService.java [hunk 1]
```diff
@@ -57,4 +68,30 @@ public ResponseEntity<BaseResponse<Object>> signUpWithEmail(SignUpWithEmailReque
...
+                userProfile.setKey(keyFullName);
+                userProfile.setValue(request.fullName());
+
+                final UserProfile profile = userProfileRepository.save(userProfile);
+
+                final UserCredential user = userVerification.getUser();
+                user.setUserInfo(List.of(profile));
+
+                final String jwtToken = jwtUtil.generateToken(user.getEmail());
+                final UserCredential credential = userCredentialRepository.save(user);
```
Comment (m-hari16) [inline src/main/java/com/bluehabit/eureka/services/SignUpService.java:126]: I didn't find encrypted password saved.

**Comment by triandamai** [comment]
@m-hari16 it's good?

**Review by m-hari16 — APPROVED** [review]
yeah, it is better

**Pull request merged**